### PR TITLE
refactor(operator): extract yamls to files

### PR DESCRIPTION
**Pull request by janisz**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6180** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6180" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/6180?utm_source=stack-comment).

File: operator/pkg/central/values/translation/translation.go [hunk 1]
```diff
@@ -195,15 +205,15 @@ func getCentralPersistenceValues(p *platform.Persistence, checker *pvcStateCheck
...
+func getCentralComponentValues(c *platform.CentralComponentSpec, m *platform.GlobalMonitoring, checker *pvcStateChecker, obsoletePVC bool) (*translation.ValuesBuilder, error) {
 	cv := translation.NewValuesBuilder()
 
 	cv.AddChild(translation.ResourcesKey, translation.GetResources(c.Resources))
 	if c.DefaultTLSSecret != nil {
 		cv.SetMap("defaultTLS", map[string]interface{}{"reference": c.DefaultTLSSecret.Name})
 	}
 
-	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled())
+	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled() || m.IsOpenShiftMonitoringEnabled())
```
Comment (stehessel) [inline operator/pkg/central/values/translation/translation.go:216]: What does this line do exactly? Note that we want to be able to configure `central.exposeMonitoring` (http endpoint, default `false`) and `monitoring.openshift.enabled` (https endpoint, default `true`) separately.

**Comment by stehessel** [comment]
I think it probably makes sense to split these changes into i) Helm chart and ii) CRD/operator.

File: operator/pkg/central/values/translation/translation.go (hunk 1 above)
Comment (janisz) [reply operator/pkg/central/values/translation/translation.go]: So this line exposes monitoring if it's enabled or if openshift monitoring that's uses exposed monitoring endpoint. I think we may add a todo here to change it once monitoring endpoint will be available.

**Comment by janisz** [comment]
> I think it probably makes sense to split these changes into i) Helm chart and ii) CRD/operator.

Good point, I'll split it into smaller PRs

**Review by stehessel — APPROVED** [review]
LGTM, but maybe wait for one review of a Draconian as well

**Review by kurlov — APPROVED** [review]

**Pull request merged**